src/helpers/mail.ts

```diff
@@ -14,10 +15,19 @@ export async function send(id: string, params: any) {
     html: compile(template.html)(params)
   };
 
-  try {
-    const result = await sgMail.send(msg);
-    console.log('email sent', result);
-  } catch (e) {
-    console.log('email failed', e);
+  if (process.env.PREVIEW === 'true') {
+    try {
+      const result: string = preview(id, msg);
+      console.log(`preview email generated at ${result}`);
+    } catch (e) {
+      console.log('preview generation failed', e);
+    }
+  } else {
+    try {
+      const result = await sgMail.send(msg);
```

Comment:
If you want to keep the test email it would be better to have it on a separate route like `send/:template/:email`